### PR TITLE
Revert "Disable event_timeupdate_noautoplay test on mac"

### PR DESCRIPTION
**Pull request by Manishearth**
Hopefully the builders now have the right gstreamer things.

From https://github.com/servo/servo/pull/21543 , needs https://github.com/servo/saltfs/pull/898

---
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/21895)

### CODE REVIEW

**Comment by Manishearth** [comment]
@bors-servo try retry

mac5 seems to have a tendency to break its virtualenv whenever you reinstall gstreamer

**Comment by ferjm** [comment]
Ugh, same failures:

```
{"status": "FAIL", "group": "default", "message": "assert_unreached: Reached unreachable code", "stack": null, "subtest": "calling play() on a sufficiently long audio should trigger timeupdate event", "test": "/html/semantics/embedded-content/media-elements/event_timeupdate_noautoplay.html", "line": 66705, "action": "test_result", "expected": "NOTRUN"}
{"status": "FAIL", "group": "default", "message": "assert_unreached: Reached unreachable code", "stack": null, "subtest": "calling play() on a sufficiently long video should trigger timeupdate event", "test": "/html/semantics/embedded-content/media-elements/event_timeupdate_noautoplay.html", "line": 66707, "action": "test_result", "expected": "NOTRUN"}
{"status": "OK", "group": "default", "message": null, "stack": null, "subtest": null, "test": "/html/semantics/embedded-content/media-elements/event_timeupdate_noautoplay.html", "line": 66708, "action": "test_result", "expected": "TIMEOUT"}
```

It seems that codecs are still not present:

```
administrators-Mac-mini:~ root# gst-inspect-1.0 | grep ogg
typefindfunctions: application/x-ogg-skeleton: no extensions
typefindfunctions: application/ogg: ogg, oga, ogv, ogm, ogx, spx, anx, axa, axv
typefindfunctions: application/x-id3v1: mp3, mp2, mp1, mpga, ogg, flac, tta
typefindfunctions: application/x-id3v2: mp3, mp2, mp1, mpga, ogg, flac, tta
libav:  avmux_ogg: libav Ogg muxer (not recommended, use oggmux instead)
```

And for some reason there are no GStreamer debug logs, even though GST_DEBUG=3 is set.

**Comment by ferjm** [comment]
I get this output trying to use playbin with the mp3 and oga tests from mac5

```
administrators-Mac-mini:~ servo$ gst-launch-1.0 playbin uri=file:///Users/servo/ferjm/servo/tests/wpt/web-platform-tests/media/sound_5.mp3
Setting pipeline to PAUSED ...
Pipeline is PREROLLING ...
Redistribute latency...
Pipeline is PREROLLED ...
Setting pipeline to PLAYING ...
New clock: GstAudioSinkClock
Got EOS from element "playbin0".
Execution ended after 0:00:05.440474000
Setting pipeline to PAUSED ...
Setting pipeline to READY ...
Setting pipeline to NULL ...
Freeing pipeline ...

administrators-Mac-mini:~ servo$ gst-launch-1.0 playbin uri=file:///Users/servo/ferjm/servo/tests/wpt/web-platform-tests/media/sound_5.
sound_5.mp3  sound_5.oga
administrators-Mac-mini:~ servo$ gst-launch-1.0 playbin uri=file:///Users/servo/ferjm/servo/tests/wpt/web-platform-tests/media/sound_5.oga
Setting pipeline to PAUSED ...
Pipeline is PREROLLING ...
Missing element: Ogg demuxer
WARNING: from element /GstPlayBin:playbin0/GstURIDecodeBin:uridecodebin0: No decoder available for type 'audio/ogg'.
Additional debug info:
gsturidecodebin.c(921): void unknown_type_cb(GstElement *, GstPad *, GstCaps *, GstURIDecodeBin *) (): /GstPlayBin:playbin0/GstURIDecodeBin:uridecodebin0
ERROR: from element /GstPlayBin:playbin0/GstURIDecodeBin:uridecodebin0: Your GStreamer installation is missing a plug-in.
Additional debug info:
gsturidecodebin.c(988): void no_more_pads_full(GstElement *, gboolean, GstURIDecodeBin *) (): /GstPlayBin:playbin0/GstURIDecodeBin:uridecodebin0:
no suitable plugins found:
gstdecodebin2.c(4640): gboolean gst_decode_bin_expose(GstDecodeBin *) (): /GstPlayBin:playbin0/GstURIDecodeBin:uridecodebin0/GstDecodeBin:decodebin0:
no suitable plugins found:
Missing decoder: Ogg (audio/ogg)

ERROR: pipeline doesn't want to preroll.
Setting pipeline to NULL ...
Freeing pipeline ...
```

This for the video tests (mp4 and ogv)

```
administrators-Mac-mini:~ servo$ gst-launch-1.0 playbin uri=file:///Users/servo/ferjm/servo/tests/wpt/web-platform-tests/media/movie_5.mp4
Setting pipeline to PAUSED ...
Pipeline is PREROLLING ...
Got context from element 'sink': gst.gl.GLDisplay=context, gst.gl.GLDisplay=(GstGLDisplay)"\(GstGLDisplayCocoa\)\ gldisplaycocoa0";
Redistribute latency...
Redistribute latency...
Got context from element 'playsink': gst.gl.GLDisplay=context, gst.gl.GLDisplay=(GstGLDisplay)"\(GstGLDisplayCocoa\)\ gldisplaycocoa0";
Redistribute latency...
Redistribute latency...
Pipeline is PREROLLED ...
Setting pipeline to PLAYING ...
New clock: GstAudioSinkClock
Got EOS from element "playbin0".
Execution ended after 0:00:05.532106000
Setting pipeline to PAUSED ...
Setting pipeline to READY ...
Setting pipeline to NULL ...
Freeing pipeline ...

administrators-Mac-mini:~ servo$ gst-launch-1.0 playbin uri=file:///Users/servo/ferjm/servo/tests/wpt/web-platform-tests/media/movie_5.ogv
Setting pipeline to PAUSED ...
Pipeline is PREROLLING ...
Missing element: Ogg demuxer
WARNING: from element /GstPlayBin:playbin0/GstURIDecodeBin:uridecodebin0: No decoder available for type 'video/ogg'.
Additional debug info:
gsturidecodebin.c(921): void unknown_type_cb(GstElement *, GstPad *, GstCaps *, GstURIDecodeBin *) (): /GstPlayBin:playbin0/GstURIDecodeBin:uridecodebin0
ERROR: from element /GstPlayBin:playbin0/GstURIDecodeBin:uridecodebin0: Your GStreamer installation is missing a plug-in.
Additional debug info:
gsturidecodebin.c(988): void no_more_pads_full(GstElement *, gboolean, GstURIDecodeBin *) (): /GstPlayBin:playbin0/GstURIDecodeBin:uridecodebin0:
no suitable plugins found:
gstdecodebin2.c(4640): gboolean gst_decode_bin_expose(GstDecodeBin *) (): /GstPlayBin:playbin0/GstURIDecodeBin:uridecodebin0/GstDecodeBin:decodebin0:
no suitable plugins found:
Missing decoder: Ogg (video/ogg)

ERROR: pipeline doesn't want to preroll.
Setting pipeline to NULL ...
Freeing pipeline ...
```

File: tests/wpt/web-platform-tests/html/semantics/embedded-content/media-elements/event_timeupdate_noautoplay.html [hunk 1]
```diff
@@ -17,7 +17,10 @@
 test(function() {
   var t = async_test("calling play() on a sufficiently long audio should trigger timeupdate event", {timeout:5000});
   var a = document.getElementById("a");
-  a.addEventListener("error", t.unreached_func());
+  a.addEventListener("error", function(e) {
+    console.log("===> ERROR", e.target.error.message, e.target.error.code);
+    t.unreached_func();
```
Comment (jdm) [inline tests/wpt/web-platform-tests/html/semantics/embedded-content/media-elements/event_timeupdate_noautoplay.html]: This is not equivalent - unreached_func returns a function. You'll want `"error", t.step_func(function(e) { ... t.assert_unreached() })`

**Comment by Manishearth** [comment]
The playbin test works now, btw (and gst-inspect shows some nice new arguments)

**Comment by Manishearth** [comment]
Wait, oops, this still has the WPT patches

**Comment by Manishearth** [comment]
All the mac wpt builds pass.

@bors-servo try- r=jdm retry

**Comment by Manishearth** [comment]
@bors-servo retry

```
{"status": "FAIL", "group": "default", "message": "/css/css-text/word-break/word-break-break-all-002.html e9f5e432dc724277e091e8b7e00f51485ab2990c\n/css/css-text/word-break/reference/word-break-break-all-ref-002.html 0a0832ae86af195ed95347e3668ce82902d13c77\nTesting e9f5e432dc724277e091e8b7e00f51485ab2990c == 0a0832ae86af195ed95347e3668ce82902d13c77", "stack": null, "subtest": null, "test": "/css/css-text/word-break/word-break-break-all-002.html", "line": 7384, "action": "test_result", "expected": "PASS"}
```

intermittent?

**Pull request merged**